weed/filer/filer.go

```diff
@@ -334,7 +334,10 @@ func (f *Filer) FindEntry(ctx context.Context, p util.FullPath) (entry *Entry, e
 	entry, err = f.Store.FindEntry(ctx, p)
 	if entry != nil && entry.TtlSec > 0 {
 		if entry.Crtime.Add(time.Duration(entry.TtlSec) * time.Second).Before(time.Now()) {
-			f.Store.DeleteOneEntry(ctx, entry)
+			err = f.doDeleteEntryMetaAndData(ctx, entry, false, false, nil)
```

Comment:
In fact, the volume server calls the CollectHeartbeat method every 5 seconds, which checks for expired volumes and deletes them. 

https://github.com/seaweedfs/seaweedfs/blob/master/weed/storage/store.go#L272-L274

The TTL mechanism for the filer server should only manage metadata and not interfere with volumes, as this could trigger a large number of chunk deletion operations, potentially leading to performance issues.

Reply:
@togettoyou I'm definitely a seaweed newbie, but I think what you're saying doesn't fix the problem liangyihan and I have? Please also see this issue: https://github.com/seaweedfs/seaweedfs/issues/5527
I mean, if I understood it right, what you linked above is an expiration check on entire volumes, *not* on single files. Or maybe I was the one misunderstanding from the beginning the -ttl option of the fs.configure command?
https://github.com/seaweedfs/seaweedfs/wiki/Path-Specific-Configuration

Reply:
@andrea-agostini My understanding is that SeaweedFS has two TTL mechanisms: one for the metadata database of the `filer server` and another for the volumes on the `volume server`. The TTL set in `fs.configure` will assign a TTL to the volume when files are uploaded. After this, the volume server will trigger the deletion of the volume through a scheduled mechanism. However, the expiration deletion of the metadata on the filer server requires a `list` or `get` operation to be triggered.

Reply:
It seems that the author of issue #5527 has already fixed it.